dspace-api/src/test/java/org/dspace/app/requestitem/CombiningRequestItemStrategyTest.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.requestitem;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+
+import java.util.List;
+
+import org.dspace.content.Item;
+import org.dspace.core.Context;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/**
+ *
+ * @author Mark H. Wood <mwood@iupui.edu>
+ */
+public class CombiningRequestItemStrategyTest {
+    /**
+     * Test of getRequestItemAuthor method, of class CombiningRequestItemStrategy.
+     * @throws java.lang.Exception passed through.
+     */
+    @Test
+    public void testGetRequestItemAuthor()
+            throws Exception {
+        System.out.println("getRequestItemAuthor");
+        Context context = null;
+
+        Item item = Mockito.mock(Item.class);
+        RequestItemAuthor author1 = new RequestItemAuthor("Pat Paulsen", "ppaulsen@example.com");
+        RequestItemAuthor author2 = new RequestItemAuthor("Alfred E. Neuman", "aeneuman@example.com");
+        RequestItemAuthor author3 = new RequestItemAuthor("Alias Undercover", "aundercover@example.com");
+
+        RequestItemAuthorExtractor strategy1 = Mockito.mock(RequestItemHelpdeskStrategy.class);
+        Mockito.when(strategy1.getRequestItemAuthor(context, item)).thenReturn(List.of(author1));
+
+        RequestItemAuthorExtractor strategy2 = Mockito.mock(RequestItemHelpdeskStrategy.class);
+        Mockito.when(strategy2.getRequestItemAuthor(context, item)).thenReturn(List.of(author2, author3));
```

Comment:
it doesn't really matter from the code perspective but it would make more sense to use the RequestItemMetadataStrategy as the test assume multiple person to be returned and talk about author. This will make the test easier to understand imho

Reply:
I agree.  Done.